File: user-backend/Common/Models/ManagementApp/Reports/UsageByUserReport.cs
using System;

namespace Common.Models.ManagementApp.Reports
{
    public class UsageByUserReport
    {
        public string CompanyName { get; set; }
        public Guid GroupId { get; set; }
        public string GroupName { get; set; }
        public string Email { get; set; }
        public int SentDocumentsCount { get; set; }
        public int SignedDocumentsCount { get; set; }
        public int DeclinedDocumentsCount { get; set; }
        public int CanceledDocumentsCount { get; set; }
        public int DeletedDocumentsCount { get; set; }
    }
}

File: wesignmanagement-client-DEV/src/app/models/results/Reports/usageByUserReport.ts
export class usageByUserReport {
    public companyName: string;
    public groupName: string;
    public email: string;
    public sentDocumentsCount: number;
    public signedDocumentsCount: number;
    public declinedDocumentsCount: number;
    public canceledDocumentsCount: number;
    public deletedDocumentsCount: number;
}

export class usageByUserReports {
    public usageByUsersReports: usageByUserReport[];
}